daos-cortx/src/samples/build_deps.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
Do we need to have this script or is there a copy in another repo? If we have a reference to original location in readme that should be sufficient. But if this automates some documented steps then its good to have this.

Reply:
contents of this script is copied from (https://github.com/hpc/mpifileutils/blob/master/doc/rst/build.rst#build-everything-directly). for a quick reference I have added this script, so that one may understand how much to copy from mentioned location.

Reply:
Script is removed and added details in setup guide.